api/Services/Mapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoMapper;
+using iTechArt.Hotels.Api.Entities;
+using iTechArt.Hotels.Api.Models;
+
+namespace iTechArt.Hotels.Api.Services
+{
+    public class Mapper : Profile
+    {
+        public Mapper()
+        {
+            CreateMap<HotelCard, HotelEntity>();
+            CreateMap<HotelEntity, HotelCard>();
```

Comment:
Is it possible to make the same differently?

Reply:
Well, it seems like I will not use mapping card to entity. I should remove this code part

Also, if I need this methods in future, I would map only not null values

Reply:
Do not completely understand what do you mean. Maybe you meant use ProjectTo nstead of Map, as Andrew said, I have changed this point

Reply:
Fix expected as a part of next PR.